### PR TITLE
update projects to target dotnet 9

### PR DESCRIPTION
**Pull request by hahn-kev**

### CODE REVIEW

**Comment by rmunn** [comment]
Curious what the benefit of moving to .NET 9 is, specifically. Is it the HybridWebView control in MAUI that we want to use? Or is this just on general principles of keeping up with the latest .NET version?

**Comment by hahn-kev** [comment]
It's just keeping up with the latest version as it comes out, in my mind we should always be on latest unless there's a specific reason not to be.

**Comment by rmunn** [comment]
.NET 9 is now officially released. It hasn't shown up in the Ubuntu repositories yet, but that's only a matter of time.

File: backend/FixFwData/FixFwData.csproj [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net8.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
+    <RootNamespace>FixFwData</RootNamespace>
+    <Description>FixFwData</Description>
+    <Company>SIL International</Company>
+    <Authors>SIL International</Authors>
+    <Product>LexBoxApi Testing</Product>
+    <Copyright>Copyright © 2023 SIL International</Copyright>
```
Comment (myieye) [inline backend/FixFwData/FixFwData.csproj:10]: I'm not sure if these tags were added on purpose, but if we keep them they should be:
```suggestion
    <Company>SIL Global</Company>
    <Authors>SIL Global</Authors>
    <Product>LexBoxApi Testing</Product>
    <Copyright>Copyright © 2024 SIL Global</Copyright>
```

**Comment by hahn-kev** [comment]
looks like our current test failure is running into this bug
https://github.com/dotnet/efcore/issues/35110
Sadly I'm not sure how to make a simple repro, so I'm going to wait and see if someone else comes up with something

**Pull request merged**